_api-files/embedding-api/documentation/authentication.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Authentication
+content-type: "embed-doc"
+order: 3
+
+sections:
+  - content: |
+      Authenticate your calls to the {{ page.api-name }} by providing an access token in your requests. Each access token is associated with a single Stitch client account. Additionally, each request's permissions are limited to that Stitch client account.
+
+      In the examples in this documentation, we use bearer auth:
+
+      ```curl
+      curl - X GET {{ page.api-base-url }}/v4/sources
+           - H "Authorization: Bearer <ACCESS_TOKEN>"
+      ```
+
+      Before you can make requests, you must register as an API client by emailing [{{ page.contact-email }}](mailto:{{ page.contact-email }}).
+
+      All requests must be made over HTTPS or they will fail. API requests that don't contain authentication will also fail.
+
+  - title: "Generate Access Tokens"
+    anchor: "generate-access-tokens"
+    content: |
+      Access tokens are obtained by performing an OAuth2 handshake with an existing Stitch client account or by creating a new account via the API.
+
+    subsections:
+      - title: "New Stitch Clients"
+        anchor: "generate-access-token-new-stitch-client"
+        content: |
+          As an API client, you can create a new Stitch client account with the Create Account API endpoint, which will return an access token for that Stitch client account.
+
+          The created account is still owned and managed by the user it is created for, and that user will be able to login to the Stitch web interface and receive emails from Stitch.
+
+      - title: "Existing Stitch Clients, Using OAuth2"
+        anchor: "existing-stitch-clients-oauth2"
+        content: |
+          You can connect to a user's existing Stitch client account by having the user complete a standard OAuth flow. Registering your application with Stitch is a prerequisite to generating tokens with OAuth, so do that first. Then, follow these steps to complete the OAuth flow:
+
+        steps:
+          - title: "Send the user to Stitch"
+            anchor: "authentication--send-user-to-stitch"
+            content: |
+              To initiate the authorization flow, the user will click a link to Stitch that includes your application's API client ID:
```

Comment:
It might be worth mentioning here somewhere that the `client_id` for OAuth is actually the `partner_id` that the user obtained when registering to use the API. 

We're keeping the client_id terminology here because it's standard OAuth terminology. Eventually, "partner" terms will be renamed to "api client" terms, and "Stitch account" / "Stitch client account" / "account" terms will be renamed to "Stitch client" terms. Not relevant for this run of the docs, but those semantic changes are coming down the line.

Reply:
Agreed. I added clarification to this particular section and the definitions listed in the table right below it.